.github/workflows/ci.yml

```diff
@@ -57,21 +57,36 @@ jobs:
           shared-key: "cargo_nextest-${{ matrix.cache_id }}"
           save-if: "${{ github.event_name == 'merge_group' }}" # save the cache only from master CI
 
-      # Run the tests
-      - run: just codecov "nextest-unit ${{ matrix.type }}"
-      - run: mv codecov.json unit-${{ matrix.id }}.json
-      - run: just codecov "nextest-integration ${{ matrix.type }}"
+      # Run the tests:
+      - run: mkdir -p coverage/profraw/{unit,integration,binaries}
```

Comment:
Out of curiosity, any reason you're creating directory at the repository root and not within the `target` where all build artifacts otherwise live? If there is none, it would perhaps be a good idea to stick with `target/` if only to avoid creating non-git-ignored files and to avoid accidental conflicts if some future change found a reason to put a `coverage` directory in the repository.

Reply:
My issue with the `target` folder is mostly that `cargo llvm-cov` searches for `*.profraw` files in it, according to rules I’m still not entirely sure about and that are definitely not documented. So I need at least one other folder to avoid it catching leftover profraws. And from then on, it makes sense to just use that other folder for everything :)